src/OpenFOAM/containers/Lists/gpuList/textures.H

```diff
@@ -84,15 +84,15 @@ inline void textures<double>::initResourceDesc(cudaResourceDesc& resDesc)
     resDesc.res.linear.desc.y = 32; 
 }
 
-#if __CUDA_ARCH__ >= 350
+//#if __CUDA_ARCH__ >= 350
```

Comment:
Could you remove all the commented code?
This will remove the unnecessary clutter while the old version of the file will still be kept in Git.